app/views/admin/settings/forms/_images.html.erb

```diff
@@ -12,22 +12,24 @@
     <div id="imagesBodyContainer" class="card-body collapse hide" aria-labelledby="imagesBodyContainer">
       <fieldset class="grid gap-4">
 
-        <%= render partial: "admin/shared/logo_upload", locals: { f: f, allowed_types: logo_allowed_types, max_file_size: logo_max_file_size, logo: Settings::General.resized_logo } %>
-
-        <div class="crayons-field">
-          <%= admin_config_label :logo_svg, "SVG logo (soon to be deprecated)" %>
-          <%= admin_config_description Constants::Settings::General::DETAILS[:logo_svg][:description] %>
-          <%= f.text_area :logo_svg,
-                          class: "crayons-textfield",
-                          value: Settings::General.logo_svg,
-                          rows: 6,
-                          placeholder: Constants::Settings::General::DETAILS[:logo_svg][:placeholder] %>
-          <% if Settings::General.logo_svg.present? %>
-            <div class="site-logo">
-              <%= Settings::General.logo_svg.html_safe %>
-            </div>
-          <% end %>
-        </div>
+        <% if FeatureFlag.enabled?(:creator_onboarding) %>
```

Comment:
When the feature flag is enabled we hide the logo_svg field behind a feature flag and show the resized_logo